pkgs/development/python-modules/ibis-framework/default.nix

```diff
@@ -60,99 +57,81 @@
 , typing-extensions
 }:
 let
-  testBackends = [ "datafusion" "duckdb" "pandas" "sqlite" ];
+  testBackends = [ "dask" "datafusion" "duckdb" "pandas" "polars" "sqlite" ];
 
   ibisTestingData = fetchFromGitHub {
     name = "ibis-testing-data";
     owner = "ibis-project";
     repo = "testing-data";
-    rev = "8a59df99c01fa217259554929543e71c3bbb1761";
-    hash = "sha256-NbgEe0w/qf9hCr9rRfIpyaH9pv25I8x0ykY7EJxDOuk=";
+    rev = "2b3968deaa1a28791b2901dbbcc9bfd3d2f23e9b";
+    hash = "sha256-q1b5IcOl5oIFXP7/P5RufncjHEVrWp4NjoU2uo/BE9U=";
   };
 in
 
 buildPythonPackage rec {
   pname = "ibis-framework";
-  version = "5.1.0";
+  version = "6.0.0";
   format = "pyproject";
 
-  disabled = pythonOlder "3.8";
+  disabled = pythonOlder "3.9";
 
   src = fetchFromGitHub {
     name = "ibis-source";
     repo = "ibis";
     owner = "ibis-project";
     rev = "refs/tags/${version}";
-    hash = "sha256-u3BBGdhWajZ5WtoBvNxmx76+orfHY6LX3IWAq/x2/9A=";
+    hash = "sha256-qIxacrRSN/Il9cY6ZsOiYLPyxI20aI+DyDM0/9jr654=";
   };
 
-  patches = [
-    # fixes a small bug in the datafusion backend to reorder predicates
-    (fetchpatch {
-      name = "fix-datafusion-compilation.patch";
-      url = "https://github.com/ibis-project/ibis/commit/009230421b2bc1f86591e8b850d37a489e8e4f06.patch";
-      hash = "sha256-5NHkgc8d2bkOMpbY1vme1XgNfyHSr0f7BrR3JTTjjPI=";
-    })
-  ];
-
   nativeBuildInputs = [
     poetry-core
+    poetry-dynamic-versioning
```

Comment:
That sounds like we maybe need to fix the python package version. Did you check if the version in the installed package is correct?

Reply:
It's correct. This dependency is roughly equivalent to depending on `setuptools-scm`. What's the problem with depending on it?